tools/dependencies/known-dependencies.txt

```diff
@@ -232,36 +236,50 @@ hadoop-yarn-common-2.7.7.jar
 hadoop-yarn-common-3.0.0.jar
 hadoop-yarn-server-common-2.6.5.jar
 hbase-annotations-2.0.0.jar
+hbase-client-1.4.3.jar
 hbase-client-2.0.0.jar
 hbase-client-2.1.0.jar
+hbase-client-2.2.3.jar
+hbase-common-1.4.3.jar
 hbase-common-2.0.0-tests.jar
 hbase-common-2.0.0.jar
 hbase-common-2.1.0.jar
+hbase-common-2.2.3.jar
 hbase-hadoop-compat-2.0.0.jar
 hbase-hadoop-compat-2.1.0.jar
+hbase-hadoop-compat-2.2.3.jar
 hbase-hadoop2-compat-2.0.0.jar
 hbase-hadoop2-compat-2.1.0.jar
+hbase-hadoop2-compat-2.2.3.jar
 hbase-http-2.0.0.jar
 hbase-http-2.1.0.jar
 hbase-mapreduce-2.0.0.jar
 hbase-mapreduce-2.1.0.jar
 hbase-metrics-2.0.0.jar
 hbase-metrics-2.1.0.jar
+hbase-metrics-2.2.3.jar
 hbase-metrics-api-2.0.0.jar
 hbase-metrics-api-2.1.0.jar
+hbase-metrics-api-2.2.3.jar
 hbase-procedure-2.0.0.jar
 hbase-procedure-2.1.0.jar
+hbase-protocol-1.4.3.jar
 hbase-protocol-2.0.0.jar
 hbase-protocol-2.1.0.jar
+hbase-protocol-2.2.3.jar
 hbase-protocol-shaded-2.0.0.jar
 hbase-protocol-shaded-2.1.0.jar
+hbase-protocol-shaded-2.2.3.jar
 hbase-replication-2.0.0.jar
 hbase-replication-2.1.0.jar
 hbase-server-2.0.0.jar
 hbase-server-2.1.0.jar
 hbase-shaded-miscellaneous-2.1.0.jar
+hbase-shaded-miscellaneous-2.2.1.jar
 hbase-shaded-netty-2.1.0.jar
+hbase-shaded-netty-2.2.1.jar
 hbase-shaded-protobuf-2.1.0.jar
```

Comment:
Cool,  but some NOTICE and License may be missing. Another question, can we unify the version, (maybe not), for example we have multiple versions of hbase.